docs/pages/setup/guides/teleport-operator.mdx

```diff
@@ -58,17 +65,17 @@ $ helm install teleport-cluster teleport/teleport-cluster \
         --set teleportVersionOverride=(=teleport.version=)
 ```
 
-This command installs the required Kubernetes CRDs and deploys the operator next to the Teleport cluster.
-All resources except the CRDs are created in the `teleport-cluster` namespace.
+This command installs the required Kubernetes CRDs and deploys the Teleport Kubernetes Operator next to the Teleport
+cluster. All resources (except the CRDs which are cluster-scoped) are created in the `teleport-cluster` namespace.
```

Comment:
```suggestion
cluster. All resources (except CRDs, which are cluster-scoped) are created in the `teleport-cluster` namespace.
```
Suggested to make it clearer that all CRDs are cluster scoped. If not, we could change to "except the CRDs that are cluster scoped".